### ISSUE TITLE
Queue does not preserve order on restart

### ISSUE THREAD

**Issue by mauleyzaola** [mauleyzaola/maupod#208]
Steps to reproduce:
* Add like 10 tracks to the queue
* Stop `maupod-player` and the backend services with `make clean`
* Start backend services with `make server` and start `maupod-player`
* Start browser with `make browser`
* Tracks are not in the original order

**Comment by mauleyzaola** [comment]
@Serle66 Please double check

**Comment by Serle66** [comment]
@mauleyzaola I followed the instructions and everything went as expected.

First, I added 11 tracks to `Queue` and play the first one.

![Screen Shot 2020-10-16 at 10 14 29 AM](https://user-images.githubusercontent.com/5943158/96276760-09936680-0f99-11eb-82d7-2ce42e5eb95c.png)

Right in the middle of the song, I stoped `maupod-player`, `browser` and server. Then, did `make clean` and restarted all services.

All tracks kept their order and `maupod-player` started playing the song right where it stoped before.

![Screen Shot 2020-10-16 at 10 15 59 AM](https://user-images.githubusercontent.com/5943158/96277037-5c6d1e00-0f99-11eb-97b5-93e9a7a35f76.png)
